app/src/main/res/values-iw/strings.xml

```diff
@@ -1,7 +1,8 @@
-<?xml version="1.0" encoding="utf-8"?><!-- Authors:
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Authors:
 * Amire80
 -->
-<resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="TypographyQuotes,TypographyEllipsis">
```

Comment:
@amire80 these issue ignorers have been deleted. I guess the only input to TW is the English translation file and if it doesn't have the `ignore` attribute then the generated files won't?
I am speculating about things I really don't know about. I don't know if a ticket needs to be opened TW side to add these to generated Hebrew translation files or if there is anything I can do on my side

Reply:
Also @Abijeet further context [here](https://github.com/kiwix/kiwix-android/issues/1711)

TL;DR we don't want to replace straight quotes with curvy quotes or `...` with the ellipsis character in hebrew

Reply:
Got it. Currently the Android exporter does not honor attributes added to the resource tag. A ticket needs to be created on our end to preserve such attributes.

For now, would it be possible to create a separate file that defines some configurations for the analysis tool? The way we have `editorconfig` files or `.eslintrc` file?

Reply:
we can specify rules for our linter in many places, I chose to put them on these xml files because I wanted them scoped to those files only, as per Amire's request.

In `AllProjectConfigurer` I have
```
lintOptions {
        isAbortOnError = true
        isCheckAllWarnings = true
        isWarningsAsErrors = true

        ignore(
          "SyntheticAccessor",
          //TODO stop ignoring below this
          "MissingTranslation",
          "CheckResult",
          "LabelFor",
          "DuplicateStrings",
          "LogConditional"
        )

        warning(
          "UnknownNullness",
          "SelectableText",
          "IconDensities",
          "ContentDescription",
          "IconDipSize"
        )
        baseline("${path}/lint-baseline.xml")
      }
```
Adding the rule here would disable the check for all files in all modules

Reply:
In the interim I can quite easily make these small updates when we do a translation update but if it can be fully automated then that is obviously preferable

Reply:
Yes, we would like them to be scoped to just that specific file. I assume it's not possible to define config for a particular language file via the `linkOptions` in `AllProjectConfigurer`?

Reply:
I can point to a `lint.xml` like [this](https://developer.android.com/studio/write/lint#sample-lint.xml-file) to specify paths to ignore, the option is not present in the `lintOptions` closure of our kotlin build plugin and I was trying to avoid spreading our lint configuration across multiple files but it seems the most viable option

Reply:
Yes, until we fix this on our end.

Reply:
Added [T244494](https://phabricator.wikimedia.org/T244494) on our end.